ui/src/views/setting/ConfigurationValue.vue

```diff
@@ -254,6 +258,17 @@ export default {
     this.setConfigData()
   },
   watch: {
+    loading (newVal, oldVal) {
```

Comment:
Is loading value for the element set somewhere in the code?
Similarly for property `configDisable`, though unrelated to the PR